### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by jane-pyc** [joonson/syncnet_python#29]
hello,
how do you define the loss function in training?
i didn't find this part.

**Comment by joonson** [comment]
This is not part of this repository. See https://github.com/joonson/syncnet_trainer
